warehouse/router/upload_stats.go

```diff
@@ -69,6 +68,17 @@ func (job *UploadJob) generateUploadSuccessMetrics() {
 		return
 	}
 
+	eventTimeRanges, err := job.stagingFileRepo.GetEventTimeRangesByUploadID(job.ctx, job.upload.ID)
+	if err != nil {
+		job.logger.Warnw("event time ranges for upload", logfield.Error, err.Error())
```

Comment:
Can you please use the non-sugared logger for better performance?

```suggestion
		job.logger.Warnn("event time ranges for upload", obskit.Error(err))
```

Example [here](https://github.com/rudderlabs/rudder-server/blob/v1.34.1/router/batchrouter/handle.go#L421).